API/Controllers/QuestionsController.cs

```diff
@@ -26,22 +26,16 @@ public async Task<ActionResult<string>> GetQuestion(int id)
 
             try
             {
-                var question = id == 0 ?
-                    await Context.Questions.Include(i => i.QuestionOptions).FirstOrDefaultAsync()
-                        .ConfigureAwait(false) :
-                    await Context.Questions.Include(i => i.QuestionOptions).SingleOrDefaultAsync(i => i.ID == id)
-                        .ConfigureAwait(false);
+                var question = await Context.Questions.Include(i => i.QuestionOptions)
```

Comment:
iirc, WebApp provides 0 as the ID when they expect to get the first question, is removing this check intended?

Reply:
You could be right, will check.

Reply:
Fixed.